apps/communications/dialer/style/call_log.css

```diff
@@ -304,3 +304,12 @@ html[dir="rtl"] .recents-edit .log-item a {
 html[dir="rtl"] #sticky {
   background-position: calc(100% - 1.5rem) 0;
 }
+
+.ellipsis-dir-fix {
+  display: block;
+  max-width: 100%;
+  width: auto;
```

Comment:
in SMS we used -moz-fit-content because otherwise in lists we could have something like this:

```
* LTR text
*                                                   RTL TEXT
* LTR text
```

and same in a RTL document:
```
                                       RTL text *
LTR text                                        *
                                       RTL text *
```

I don't know if you have such cases in your app.